receiver/gitproviderreceiver/internal/scraper/githubscraper/genqlient.graphql

```diff
@@ -36,6 +36,41 @@ query checkLogin($login: String!) {
   }
 }
 
+query getCommitData(
+  $name: String!
+  $owner: String!
+  $branchFirst: Int!
+  $commitFirst: Int!
+  # @genqlient(pointer: true)
+  $commitCursor: String
+  $branchName: String!
+) {
+  repository(name: $name, owner: $owner) {
+    refs(refPrefix: "refs/heads/", first: $branchFirst, query: $branchName) {
+      # @genqlient(typename: "BranchHistory")
+      nodes {
+        target {
+          ... on Commit {
+            id
+            history(first: $commitFirst, after: $commitCursor) {
+              # @genqlient(typename: "CommitNode")
```

Comment:
@crobert-1 this is the query we use that gets populated into generated Go code. Instead of uses `edges` for commits, I adjusted the query and shortened the name. We still do a type check because of the `... on Commit` syntax on the query.